force-app/main/default/flows/Enter_Journey_Details.flow-meta.xml

```diff
@@ -0,0 +1,639 @@
+        <label>Unchanged Task Loop</label>
+        <locationX>402</locationX>
+        <locationY>674</locationY>
+        <collectionReference>colTemplateTasks</collectionReference>
+        <iterationOrder>Asc</iterationOrder>
+        <nextValueConnector>
+            <targetReference>Assign_Unedited_Task_Details</targetReference>
+        </nextValueConnector>
+        <noMoreValuesConnector>
+            <targetReference>Create_Unedited_Journey_Tasks</targetReference>
+        </noMoreValuesConnector>
+    </loops>
+    <processMetadataValues>
+        <name>BuilderType</name>
+        <value>
+            <stringValue>LightningFlowBuilder</stringValue>
+        </value>
+    </processMetadataValues>
+    <processMetadataValues>
+        <name>CanvasMode</name>
+        <value>
+            <stringValue>AUTO_LAYOUT_CANVAS</stringValue>
+        </value>
+    </processMetadataValues>
+    <processMetadataValues>
+        <name>OriginBuilderType</name>
+        <value>
+            <stringValue>LightningFlowBuilder</stringValue>
+        </value>
+    </processMetadataValues>
+    <processType>Flow</processType>
+    <recordCreates>
+        <description>Creates new Journey Task records by using the collection containing the edited Journey Tasks (which were previously Template Tasks).</description>
+        <name>Create_Journey_Tasks_from_Template</name>
+        <label>Create Journey Tasks from Template</label>
+        <locationX>50</locationX>
+        <locationY>1298</locationY>
+        <connector>
+            <targetReference>View_New_Journey</targetReference>
+        </connector>
+        <inputReference>colJourneyTasks</inputReference>
+    </recordCreates>
+    <recordCreates>
+        <description>This element creates a new Journey record using the details provided by the user, such as the Journey Name, Candidate Information, Manager Information, and Onboarding Start Date, linking it to the Template from which the Flow was launched.</description>
+        <name>Create_New_Journey</name>
+        <label>Create New Journey</label>
+        <locationX>226</locationX>
+        <locationY>458</locationY>
+        <connector>
+            <targetReference>Edit_Tasks</targetReference>
+        </connector>
+        <inputAssignments>
+            <field>Candidate_Email__c</field>
+            <value>
+                <elementReference>Candidate_Email.value</elementReference>
+            </value>
+        </inputAssignments>
+        <inputAssignments>
+            <field>Candidate_Name__c</field>
+            <value>
+                <elementReference>Candidate_Name</elementReference>
+            </value>
+        </inputAssignments>
+        <inputAssignments>
+            <field>Manager_Email__c</field>
+            <value>
+                <elementReference>Manager_Email.value</elementReference>
+            </value>
+        </inputAssignments>
+        <inputAssignments>
+            <field>Manager_Name__c</field>
```

Comment:
I can put an email address in this field, should I be able to? I know Elon Musk's child is called X Æ A-Xii but in reality most people don't have email addresses as names.

It's 2023 so maybe I'm just behind the time

Reply:
Solved this by using a REGEX expression to not allow numbers or special characters.